resources/frameworks_2021Q3.yaml

```diff
@@ -0,0 +1,91 @@
+---
+
+#########################
+### AutoML frameworks ###
+#########################
+
+AutoGluon:
+  version: "0.3.1"
+
+AutoGluon_benchmark:
+  extends: AutoGluon
+  description: "AutoGluon with 'best_quality' preset provides the most accurate overall predictor"
+  params:
+    presets: best_quality
+
+autosklearn:
+  version: '0.13.0'
+
+autosklearn2:
+  extends: autosklearn
```

Comment:
we'll run both, correct?

Reply:
That's the plan, though v2 does not support regression and does not have clean meta-learning yet - so in terms of results we'll need to figure out a suitable way to represent them.